app/admin/allowed_user_email_masks.rb

```diff
@@ -0,0 +1,6 @@
+ActiveAdmin.register AllowedUserEmailMasks do
+  menu :priority => 100
```

Comment:
The new Ruby syntax is `priority: 100` instead of `:priority => 100`.
Does it work here and everywhere else you added a `=>`?